app/src/main/res/values/strings.xml

```diff
@@ -714,10 +714,16 @@
         When the paperclip button gets hidden to provide more space for the input field,
         you can still attach files via overflow menu.</string>
 
-    <string name="pref__buttons__volume_buttons_change_text_size">
-        Volume buttons change text size</string>
-    <string name="pref__buttons__volume_buttons_change_text_size_summary">
-        If set, volume buttons will change text size instead of volume</string>
+    <string name="pref__buttons__volume__title">Volume button role</string>
```

Comment:
I would prefer just `Volume buttons`, so that the preference reads like a sentence, that is `Volume buttons ... Change text size`, etc. Our other preferences are modeled like this

Reply:
Good idea. Done.